app/middleware/registerIncomingService.js

```diff
@@ -67,6 +69,10 @@ const validateParameters = req => {
         logger.error(`Service ${req.query.serviceId} is not registered with PCQ`);
     } else {
         logger.info('Parameters verified successfully.');
+        const instrumentationKey = config.get('appInsights.instrumentationKey');
```

Comment:
why don't you do this in the app insights module? and then anywhere you would like to track something, you'd only log a message.